application/models/models.py

```diff
@@ -25,12 +28,10 @@ class ClassificationType(enum.Enum):
 class CommunicationTemplate(Base):
```

Comment:
Does this need a `__table_args__` attribute? One that specifies a non-public db schema would be useful.